packages/components/src/EditableText/EditableText.component.js

```diff
@@ -9,30 +9,22 @@ import theme from './EditableText.scss';
 import I18N_DOMAIN_COMPONENTS from '../constants';
 
 function PlainTextTitle({ onEdit, disabled, text, inProgress, t }) {
+	const isDisabled = disabled || inProgress;
 	return (
-		<div>
-			<button
-				className={classNames(
-					theme['tc-editable-text-text-wording-button'],
-					'tc-editable-text-text-wording-button',
-					'btn',
-					'btn-link',
-				)}
-				onDoubleClick={onEdit}
-				disabled={disabled || inProgress}
+		<div className={theme['tc-editable-text-title']}>
+			<span
+				className={classNames(theme['tc-editable-text-wording'], 'tc-editable-text-wording')}
+				onDoubleClick={isDisabled ? undefined : onEdit}
 			>
 				{text}
-			</button>
+			</span>
 			<Action
 				name="action-edit"
-				label={t('MODIFY_TOOLTIP', { defaultValue: 'Edit' })}
+				label={t('EDITABLE_TEXT_EDIT', { defaultValue: 'Edit' })}
```

Comment:
You're right :/
But this key doesn't follow any format we use in other component. It doesn't provide any context. So when translators will ask where it is used, even the devs won't even know without searching in the whole codebase.
I propose to keep that, it will break 1 time, but will help us in the future. I'll document that.

Reply:
Reverted